types/react-leaflet/v1/index.d.ts

```diff
@@ -140,18 +145,40 @@ export interface MapProps extends MapEvents, Leaflet.MapOptions, Leaflet.LocateO
     minZoom?: number;
     style?: React.CSSProperties;
     useFlyTo?: boolean;
+    viewport?: Viewport;
+    whenReady?: () => void;
     zoom?: number;
 }
 
 export class Map<P extends MapProps = MapProps, E extends Leaflet.Map = Leaflet.Map> extends MapComponent<P, E> {
     className?: string;
     container: HTMLDivElement;
+    viewport: Viewport;
     getChildContext(): { layerContainer: E, map: E };
     createLeafletElement(props: P): E;
     updateLeafletElement(fromProps: P, toProps: P): void;
     bindContainer(container: HTMLDivElement): void;
     shouldUpdateCenter(next: Leaflet.LatLngExpression, prev: Leaflet.LatLngExpression): boolean;
     shouldUpdateBounds(next: Leaflet.LatLngBoundsExpression, prev: Leaflet.LatLngBoundsExpression): boolean;
+    onViewportChange: (viewport: Viewport | null) => void;
```

Comment:
These don't need `| null`

Reply:
Fixed. This was copied from the v2 types but I think that's also wrong because the method on the Map component class itself does not take in any parameters https://github.com/PaulLeCam/react-leaflet/blob/v1/src/Map.js#L283